cluster/ampagent/cmd/install.go

```diff
@@ -326,43 +327,120 @@ func createInitialNetworks() error {
 func removeInitialNetworks() error {
 	for _, network := range ampnetworks {
 		// Check if network already exists
-		exists, err := Docker.NetworkExists(network)
+		id, err := Docker.NetworkID(network)
 		if err != nil {
 			return err
 		}
-		if !exists {
-			log.Println("Skipping non existing network:", network)
-			return nil
+		if id == "" {
+			// Skipping non existent network
+			continue
 		}
 
 		// Remove network
-		time.Sleep(2 * time.Second)
-		if err := Docker.RemoveNetwork(network); err != nil {
+		if err := Docker.RemoveNetwork(id); err != nil {
 			return err
 		}
-		log.Println("Successfully removed network:", network)
+		log.Printf("Successfully removed network %s [%s]", network, id)
+	}
+	return nil
+}
+
+func removeExitedContainers(timeout int) error {
+	i := 0
+	dontKill := []string{"amp-agent", "amp-local"}
+	var containers []types.Container
+	if timeout == 0 {
+		// default value
```

Comment:
Prefer comment to the right of the instruction for readability:
`timeout = 30 // default timeout`